i18n/pl/docusaurus-plugin-content-docs/current/learn/grammar/nouns.md

```diff
@@ -62,7 +62,7 @@ Rzeczowniki rodzaju nijakiego kończą się na `-o` (rdzenie twarde) lub `-e` (r
 - Na mianownik/biernik i instrumentalną liczbę pojedynczą wpływa zasada [`o`/`e` ][4].
 - Jeśli chodzi o miejscownik liczby pojedynczej, to, co dotyczy rzeczowników rodzaju męskiego, dotyczy także rzeczowników rodzaju nijakiego: zamiast `-u` można również napisać `-ě` po twardej spółgłosce lub `-i` po miękkiej spółgłosce.
 - Rzeczowniki rodzaju nijakiego nie mają wołacza odrębnego od mianownika.
-- W dopełniaczu liczby mnogiej rzeczowniki rodzaju nijakiego pierwszej deklinacji mają zerową końcówkę `-Ø`. W przypadkach, gdy prowadzi to do niemożliwych skupień spółgłosek, `-e-` jest wstawiane przed `-j` lub po miękkiej spółgłosce, albo `-o-` między twardymi spółgłoskami: :isv[okn**o**] → :isv[ok**o**n], :isv[morj**e**] → :isv[mor**e**j].
+- W dopełniaczu liczby mnogiej rzeczowniki rodzaju nijakiego pierwszej deklinacji mają końcówkę `-Ø`. W przypadkach, gdy prowadzi to do niemożliwych zbitek spółgłosek, `-e-` jest wstawiane przed `-j` lub po miękkiej spółgłosce, albo `-o-` między twardymi spółgłoskami: :isv[okn**o**] → :isv[ok**o**n], :isv[morj**e**] → :isv[mor**e**j].
```

Comment:
@Deczski či bude směsta srazumělo že `-Ø` to je odsutstvo koncovky, a ne někaky specialny simbol, ktory bude zamyliti čitatelja?

Reply:
```suggestion
- W dopełniaczu liczby mnogiej rzeczowniki rodzaju nijakiego pierwszej deklinacji mają zerową końcówkę `-Ø`. W przypadkach, gdy prowadzi to do niemożliwych zbitek spółgłosek, `-e-` jest wstawiane przed `-j` lub po miękkiej spółgłosce, albo `-o-` między twardymi spółgłoskami: :isv[okn**o**] → :isv[ok**o**n], :isv[morj**e**] → :isv[mor**e**j].
```